### ISSUE TITLE
Minikube fails to start start on Hyper-V with Windows Server 2016 or Windows 10 due to KVP issues

### ISSUE THREAD

**Issue by alexpilotti** [kubernetes/minikube#1598]
**Is this a BUG REPORT or FEATURE REQUEST?** (choose one): BUG REPORT

**Minikube version** (use `minikube version`): v0.19.1

**Environment**: Hyper-V
- **OS** (e.g. from /etc/os-release): Windows Server 2016
- **VM Driver** (e.g. `cat ~/.minikube/machines/minikube/config.json | grep DriverName`): hyperv
- **ISO version** (e.g. `cat ~/.minikube/machines/minikube/config.json | grep -i ISO` or `minikube ssh cat /etc/VERSION`): minikube-v0.18.0.iso
- **Install tools**:
- **Others**:

**What happened**:

When starting Minikube, it will hang waiting for retrieving an IP address via KVP (e.g. Get-VMNetworkAdapter minikube). An IP address is correctly assigned in the VM, this can be verified by logging in via ssh using the docker / tcuser credentials.

The following messages are visible in the "minikube start" verbose output:

```
(( Get-VM minikube ).networkadapters[0]).ipaddresses[0]
(minikube) DBG | [stdout =====>] :
(minikube) DBG | [stderr =====>] :
(minikube) DBG | [executing ==>] : C:\Windows\System32\WindowsPowerShell\v1.0\powershell.exe -NoProfile -NonInteractive
( Get-VM minikube ).state
```

This issue is due to outdated LIS modules / KVP daemon in the ISO.

**What you expected to happen**:

minikube should retrieve an IP address and proceed with the deployment.

**How to reproduce it** (as minimally and precisely as possible):

`.\minikube.exe start --vm-driver=hyperv --hyperv-virtual-switch=<your_external_switch> -v 9`

**Anything else do we need to know**:

This issue is common on any Linux distro and can be fixed by updating the Kernel used in the ISO. This will include recent LIS modules that fix the issue.

**Comment by alexpilotti** [comment]
Update: building a [minikube ISO from source](https://github.com/kubernetes/minikube/blob/master/docs/contributors/minikube_iso.md) and using it with "minikube start" works:
 
```
.\minikube.exe start --vm-driver=hyperv --hyperv-virtual-switch=External --iso-url=https://<your_url_here>/minikube.iso -v 9
```
The Linux kernel version now is 4.9.13, which includes the required LIS fixes.

Would it be possible to release a new default ISO?

**Comment by r2d4** [comment]
@alexpilotti we've released a 0.19.1 image, however there hasn't been a minikube release with it yet
It will be the default in the next version of minikube.

`https://storage.googleapis.com/minikube/iso/minikube-v0.19.1.iso`

**Comment by alexpilotti** [comment]
@r2d4 great thanks, I just tested it performing multiple start / delete runs and I noticed a race condition behaviour that causes failures in some cases. 

Is there already a plan to switch to BUILDROOT_BRANCH 2017.5 in the ISO? The kernel version is 4.11.5.
